shell/packages/sandstorm-db/db.js

```diff
@@ -938,6 +974,21 @@ _.extend(SandstormDb.prototype, {
 
 if (Meteor.isServer) {
   SandstormDb.prototype.getWildcardOrigin = getWildcardOrigin;
+
+  const Crypto = Npm.require("crypto");
+  SandstormDb.prototype.removeApiTokens = function (query) {
```

Comment:
I wish there were a way to neuter `ApiTokens.remove()`/`db.collections.apiTokens.remove()`, so it'd be impossible for new code to get this wrong.  Perhaps we can monkeypatch:

```js
ApiTokens.removeUnchecked = ApiTokens.remove;
ApiTokens.remove = function () { throw new Error("Use db.removeApiTokens") }
```

and then use `ApiTokens.removeUnchecked()` in this function.  Thoughts?